board/common/rootfs/usr/libexec/infix/init.d/20-nameif

```diff
@@ -30,12 +30,16 @@ for netif in /sys/class/net/*; do
 done
 
 # Find and mark all switch ports
-ports=$(devlink -j port | jq -r '.port[]
-				 | select(.flavour == "physical")
-				 | .netdev')
-for iface in $ports; do
+if command -v devlink > /dev/null 2>&1; then
```

Comment:
Do we know why `devlink` is not installed on these targets? Presumably the same version of `iproute2` is used as on `aarch64_deconfig`

Reply:
Good point, I realize now it actually is installed, so it must be the kernel side that is missing here too.

Reply:
Looked at the wrong target (the r2s).  This fix was for a log warning I ran into on `x86_64*_defconfig` -- it does not have `devlink` because it doesn't have LIBMNL enabled.  LIBMNL is selected on r2s and other aarch64 targets by the mdio-tools package, which we don't select on x86_64 ... sneaky.

So we should probably just add LIBMNL to the x86_64 targets.